examples/siro_sandbox/server/remote_gui_input.py

```diff
@@ -0,0 +1,262 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Meta Platforms, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+import magnum as mn
+from server.average_rate_tracker import AverageRateTracker
+
+from habitat.gui.gui_input import GuiInput
+
+
+class RemoteGuiInput:
+    def __init__(self, interprocess_record, debug_line_render):
+        self._recent_client_states = []
+        self._interprocess_record = interprocess_record
+        self._debug_line_render = debug_line_render
+
+        self._receive_rate_tracker = AverageRateTracker(2.0)
+
+        # temp hack
+        self._key_map = {
+            "q": GuiInput.KeyNS.Q,
+            "w": GuiInput.KeyNS.W,
+            "e": GuiInput.KeyNS.E,
+            "a": GuiInput.KeyNS.A,
+            "s": GuiInput.KeyNS.S,
+            "d": GuiInput.KeyNS.D,
+            "f": GuiInput.KeyNS.F,
+        }
+
+        # temp map VR button to key
+        self._button_map = {
```

Comment:
This is pretty hacky. As a container, `GuiInput` currently only supports keyboard keys and mouse buttons (not VR controller buttons), so I map VR buttons to certain keyboard keys.